content/docs/searching.md

```diff
@@ -1339,23 +1339,26 @@ Next will be a **1702** message that contains a bit-weighted number in **lParam*
-| 128             | 0010000000        | In selection                       |
-| 256             | 0100000000        | Wrap around                        |
-| 512             | 1000000000        | Backward direction (*)             |
-
-*: **Backward direction** checked means 512 is _not_ included; unchecked means 512 _is_ included.
+| 1702-Bit-Weight |Binary-Bit-Weight      | Meaning (equivalent option checked) | Alternate Meaning (\*\*) |
+|----------------:|----------------------:|:------------------------------------|:-----------------------|
+| 1               | 0000-0000-0001        | Match whole word only               |                        |
+| 2               | 0000-0000-0010        | Match case                          |                        |
+| 4               | 0000-0000-0100        | Purge for each search               |                        |
+| 16              | 0000-0001-0000        | Bookmark line                       |                        |
+| 32              | 0000-0010-0000        | In all sub-folders                  |                        |
+| 64              | 0000-0100-0000        | In hidden folders                   |                        |
+| 128             | 0000-1000-0000        | In selection                        | Project Panel 1 (\*\*)   |
+| 256             | 0001-0000-0000        | Wrap around                         | Project Panel 2 (\*\*)   |
+| 512             | 0010-0000-0000        | Backward direction (\*)              | Project Panel 3 (\*\*)    |
+| 1024            | 0100-0000-0000        | . matches newline                   |                        |
+
+\*: **Backward direction** checked means 512 is _not_ included; unchecked means 512 _is_ included.
+
+\*\*: **Project Panel "alternate meaning"** column shows the meaning for those those bits when the action **1701** message (below) is set to **Find All** (in Projects) or **Replace in Projects**.
 
 > Let's see how the example value 515 used above is decoded:
 
-> lParam="515" (decimal) = 203 (hex) = 10 0000 0011 (binary) = 512 + 2 + 1 = (***not*** Backward direction + Match case + Match whole word only).  Thus, this would represent a forward-from-caret-towards-end-of-file search of exact case specified, with the additional qualifier that the match text must be bracketed by non-word characters.
+> lParam="515" (decimal) = 203 (hex) = 0010 0000 0011 (binary) = 512 + 2 + 1 = (***not*** Backward direction + Match case + Match whole word only).  Thus, this would represent a forward-from-caret-towards-end-of-file search of exact case specified, with the additional qualifier that the match text must be bracketed by non-word characters.
 
 
 Finally appears a **1701** message which encodes the Find family action to perform in **lParam**, which, when executed will conduct the action using all of the information encoded in the prior messages; let's do a **Replace in Files**, which has an integer value of 1660, for purposes of an example:
```

Comment:
Finally, a 1701 message appears

Reply:
The passive "appears" bothers me (both in the weird-order original and in your rework).

I am going to use "The last message in the sequence is a **1701** message, which..."